### PR TITLE
Minutes from CG plenary 2024-02-28

### PR DESCRIPTION
**Pull request by michielbdejong**

### CODE REVIEW

File: meetings/2024-02-28.md
Comment (csarven) [inline meetings/2024-02-28.md:7]:
```suggestion
Status: Published

```

File: meetings/2024-02-28.md
Comment (csarven) [inline meetings/2024-02-28.md:78]:
```suggestion
* ML: We're making SPARQL queries with Communica. Fetch metadata index, then know where to go for, e.g., skills, cities. Then from the skill index, then continue to find the appropriate index with defined `ex:forProperty`, `ex:forValue`. Trying to make generic vocab for indexing. And also `rdfs:seeAlso`.
```

File: meetings/2024-02-28.md
Comment (csarven) [inline meetings/2024-02-28.md:115]:
```suggestion
* TBL: Agree. Need to make sure the Solid community understands. People designing stuff for nuclear warheads.. ??? or someone linked to their online profiles. CC would give a token number, where it would work only once. Could that with WebID. Go to restaurant, happy to authn as me, but also come up with my phone and have it generate a ephemeral ID where I'd only give my address. For whistleblowers, for example, where they dump some information.
```

File: meetings/2024-02-28.md
Comment (TallTed) [inline meetings/2024-02-28.md:52]:
```suggestion
RG: Prof Lixia Zhang (UCLA) is organizing NDNComm at [NIST](https://www.nist.gov) with a panel titled "From local-first to fully-decentralized applications"
RG: Description: There exist many ongoing efforts aiming to steer Internet applications towards decentralized realizations; one such work was presented at last IETF — "Local-first software, resilient and secure collaboration".  This panel will showcase examples of decentralized designs; articulate their commonalities and differences; share experience from the different efforts; and identify remaining challenges in building secure, resilient, and easy to use and deploy codebase with well defined API to facilitate new generations of decentralized apps.
RG: Would anyone in this meeting be interested in, or know someone who would be interested in, speaking about Solid and, especially, Linked Data, at this Panel? Please get in touch with me over chat. 
RG: Everyone is invited to join as participants.
```

**Review by TallTed — CHANGES_REQUESTED** [review]

File: meetings/2024-02-28.md
Comment (michielbdejong) [reply meetings/2024-02-28.md:81]: It's actually spelt like that: https://comunica.dev/about/

**Comment by michielbdejong** [comment]
@VirginiaBalseiro can you merge this and/or give me permission to do so?

Screenshot 2024-03-01 at 10 22 02

I think I'll be allowed to merge it once at least one committer approves it.

**Comment by VirginiaBalseiro** [comment]
> @VirginiaBalseiro can you merge this and/or give me permission to do so?
> I think I'll be allowed to merge it once at least one committer approves it.

We all have the same permission. If you have no approvals you can tick the Merge without waiting for requirements box and merge, like I usually do.

**Review by csarven — APPROVED** [review]
Good catch re Comunica. I thought I misspelled it at first =)

Please squash merge at your convenience if no other suggestions.

**Comment by elf-pavlik** [comment]
> If you have no approvals you can tick the *Merge without waiting for requirements...*

The screenshot shared by @michielbdejong doesn't seem to have that option, it says that he is not authorized.

**Comment by csarven** [comment]
This repository has branch protection requiring one approval for PRs. I've just double checked. The chairs should now be able to bypass the branch protection rules in addition to dismissing PRs. Chairs already have Write access to the repository.

On a relate note, earlier today I've also updated https://github.com/w3c-cg/solid/pull/17 to indicate PR review time for minutes.

File: meetings/2024-02-28.md
Comment (TallTed) [reply meetings/2024-02-28.md:81]: It's worth linking.
```suggestion
* ML: We're making SPARQL queries with [Comunica](https://comunica.dev/about/). Fetch metadata index, then know where to go for, e.g., skills, cities. Then from the skill index, then continue to find the appropriate index with defined `ex:forProperty`, `ex:forValue`. Trying to make generic vocab for indexing. And also `rdfs:seeAlso`.
```

**Pull request merged**
